package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "pageres",
-	"version": "5.2.1",
+	"version": "5.3.0",
```

Comment:
Don't bump a package version in a PR. This applies in general in open source. It's up to the maintainer when and what version to use.